arelle/CntlrCmdLine.py

```diff
@@ -242,6 +242,8 @@ def parseArgs(args):
                       help=_("Write log file mode, a=append if existing (default), w=overwrite if existing. "))
     parser.add_option("--logFormat", "--logformat", action="store", dest="logFormat",
                       help=_("Logging format for messages capture, otherwise default is \"[%(messageCode)s] %(message)s - %(file)s\"."))
+    parser.add_option("--logJsonMaxAttributeLength", "--logjsonmaxattributelength", action="store", dest="logJsonMaxAttributeLength", type="int",
+                      help=_("Truncate JSON log file attribute values at length. The default is 4096000."))
```

Comment:
This isn't quite right. It's still an XML file that's being logged to. The difference is that the content of the message is json. I don't think it's worth cluttering the CLI with two options. If a user needs to increase this they can use the other option.

I think the best course of action is to use the defaults (128/4096000) if the user doesn't provide their own value, but if they do, use it for all messages.